doc/release/1.7.0-notes.rst

```diff
@@ -24,53 +24,289 @@ For running on PyPy, PyPy3 6.0+ is required.
+``nbinom`` now have Boost backends, and it is straightforward to leverage
+the backend for additional functions.
+
+The skew Cauchy probability distribution has been implemented as
+`scipy.stats.skewcauchy`.
+
+The Zipfian probability distribution has been implemented as
+`scipy.stats.zipfian`.
+
+The new distributions ``nchypergeom_fisher`` and ``nchypergeom_wallenius``
+implement the Fisher and Wallenius versions of the noncentral hypergeometric
+distribution, respectively.
+
+The generalized hyperbolic distribution was added in
+`scipy.stats.genhyperbolic`.
+
+`scipy.stats.argus` now has improved handling for small parameter values.
+
+Better argument handling/preparation has resulted in performance improvements
+for many distributions.
+
+The ``cosine`` distribution has added ufuncs for ``ppf``, ``cdf``, ``sf``, and
+``isf`` methods including numerical precision improvements at the edges of the
+support of the distribution.
+
+An option to fit the distribution to data by the method of moments has been
+added to the ``fit`` method of the univariate continuous distributions.
+
+Other
+-----
+`scipy.stats.bootstrap` has been added to allow estimation of the confidence
+interval and standard error of a statistic.
+
+The new function `scipy.stats.contingency.crosstab` computes a contingency
+table (i.e. a table of counts of unique entries) for the given data.
+
+New `scipy.stats.qmc` module
+----------------------------
+
+This new module provides Quasi-Monte Carlo (QMC) generators and associated
+helper functions.
+
+It provides a generic class `scipy.stats.qmc.QMCEngine` which defines a QMC
+engine/sampler. An engine is state aware: it can be continued, advanced and
+reset. 3 base samplers are available:
+
+- `scipy.stats.qmc.Sobol` the well known Sobol low discrepancy sequence.
+  Several warnings have been added to guide the user into properly using this
+  sampler. The sequence is scrambled by default.
+- `scipy.stats.qmc.Halton`: Halton low discrepancy sequence. The sequence is
+  scrambled by default.
+- `scipy.stats.qmc.LatinHypercube`: plain LHS design.
+
+And 2 special samplers are available:
+
+- `scipy.stats.qmc.MultinomialQMC`: sampling from a multinomial distribution
+  using any of the base `scipy.stats.qmc.QMCEngine`.
+- `scipy.stats.qmc.MultivariateNormalQMC`: sampling from a multivariate Normal
+  using any of the base `scipy.stats.qmc.QMCEngine`.
+
+The module also provide the following helpers:
+
+- `scipy.stats.qmc.discrepancy`: assess the quality of a set of points in terms
+  of space coverage.
+- `scipy.stats.qmc.update_discrepancy`: can be used in an optimization loop to
+  construct a good set of points.
+- `scipy.stats.qmc.scale`: easily scale a set of points from (to) the unit
+  interval to (from) a given range.
+
+_We gratefully acknowledge the Chan-Zuckerberg Initiative Essential Open Source
+Software for Science program for supporting many of these improvements to
```

Comment:
```suggestion
*We gratefully acknowledge the Chan-Zuckerberg Initiative Essential Open Source
Software for Science program for supporting many of these improvements to*
```
Ralf suggested before that this be italicized; I did the underscores because I was thinking of markdown, but I think single asterisk should do it.